<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rasterBandAnalysisClass</class>
 <widget class="QWidget" name="rasterBandAnalysisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProcessRaster</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLineEdit" name="lineEditGrid">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输入栅格：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openFile">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/openFile.png</normaloff>:/login/res/openFile.png</iconset>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;将对输入的栅格数据进行如下操作：&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;① 真彩色显示；&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;② 假彩色显示；&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;③ 绘制灰度直方图；&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;④ 直方图均衡化增强显示。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;请选择您需要的结果，并选择相应序号的路径。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="begin">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>370</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditOutputRaster1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出栅格 ①：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditOutputRaster2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出栅格 ②：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditOutputRaster3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出栅格 ③：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="outputRasterButton1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="outputRasterButton2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>240</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="outputRasterButton3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditOutputRaster4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出栅格 ④：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="outputRasterButton4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>320</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="login.qrc"/>
 </resources>
 <connections/>
</ui>
